aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -999,9 +1030,30 @@ module AptosFramework::Stake {
             Option::is_some(&find_validator(&validator_set.pending_active, pool_address))
     }
 
+    fun validate_required_stake(minimum_stake: u64, maximum_stake: u64) {
+        assert!(minimum_stake <= maximum_stake && maximum_stake > 0, Errors::invalid_argument(EINVALID_STAKE_RANGE));
```

Comment:
I think those functions would be actually better inlined and that you "validate the required stake" is then in a comment. Unless you are actually having (or strongly expecting) the same logic used from elsewhere. Even than such range checks might be inlined.

Reply:
Yes this is currently used twice in update_required_stake and initialize_validator_set. It's abstracted into a separate function in case we need to add more rules around required stake ranges.